Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.22-alpine AS builder
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+
+RUN go mod download
+RUN go mod verify
+
+COPY ./ ./
+
+RUN go build -ldflags="-w -s" -o server ./cmd
+
+FROM scratch
+COPY --from=builder app/server /usr/server
+
+ENTRYPOINT [ "/usr/server" ]
```

Comment:
nit but I would either put it into / or in /usr/bin or /usr/local/bin but putting a binary where usually only sub directories exist is not what unix ppl (like me) expect